src/CentralRouterInterfaceNode.cpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file CentralRouterInterfaceNode.cpp
+ *
+ * This is part of the DUNE DAQ Software Suite, copyright 2022.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#include "dtpcontrols/CentralRouterInterfaceNode.hpp"
+
+UHAL_REGISTER_DERIVED_NODE(CentralRouterInterfaceNode)
+
+CentralRouterInterfaceNode::CentralRouterInterfaceNode(const uhal::Node &node)
+    : DTPNode(node) {}
+
+CentralRouterInterfaceNode::~CentralRouterInterfaceNode() {}
+
+void CentralRouterInterfaceNode::EnableRouterInterface(bool dispatch) {
+  try {    
+    getNode("csr.ctrl.en").write(0x1);
+  } catch (...) {
```

Comment:
why do you catch everything here?  i don't think it's required that this function guarantee no throw, so this should not be here (i think).

in fact, try-catch is rarely used in timing sys code.  so why do you try at all ?

Reply:
changed 13854d4